services-js/commissions-app/src/client/common/Checkboxes.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Since this is just one checkbox, name this file `Checkbox.tsx` not `Checkboxes.tsx`.

Reply:
This file didn't get deleted. Use `git add -u` to update all files that Git knows about to capture deletions. `git add .` doesn’t add any deletions because it only matches files that currently exist.